app/static/js/functions.js

```diff
@@ -20,7 +20,8 @@ function getCoursesFromServer(subject, professor, check100, check200, check300,
             displayedCoursesList.innerHTML = null;
             var newDisplayedCoursesList = "";
             for(var i=0; i<coursesInSideBar.length; i++){
-                newDisplayedCoursesList += "<li><a href=\"#\" onclick='addCourseToCalendar(this)'>" + coursesInSideBar[i]["title"] + "</a></li>\n";
+                newDisplayedCoursesList += "<li><a href=\"#\" onclick='addCourseToCalendar(this)'>" +
+                    coursesInSideBar[i]["title"] + "</a></li>\n";
```

Comment:
Did you add this newline on purpose to make it more readable, or was this just an accidental enter smack?

Reply:
I don't remember doing it.  I'll change it back to avoid weirdness.